lib/remote/apilistener.cpp

```diff
@@ -1758,6 +1802,24 @@ std::set<JsonRpcConnection::Ptr> ApiListener::GetAnonymousClients() const
 	return m_AnonymousClients;
 }
 
+void ApiListener::DisconnectJsonRpcConnections()
+{
+	for (auto endpoint : ConfigType::GetObjectsByType<Endpoint>()) {
+		if (endpoint->GetConnected()) {
+			for (const auto& client : endpoint->GetClients()) {
+				client->Disconnect();
+			}
+		}
```

Comment:
The additional `if` check around the loop can be omitted. If `GetConnected()` returned false, `GetClients()` will return an empty set anyway.